framework/platform/android/android_platform.cpp

```diff
@@ -295,7 +295,8 @@ inline TouchAction translate_touch_action(int action)
 	return TouchAction::Unknown;
 }
 
-void on_content_rect_changed(ANativeActivity *activity, const ARect *rect)
+// TODO: update the content rect
```

Comment:
does this `TODO` still apply?

Reply:
This one is not needed, I will removed it for sure.